internal/pkg/show/config/plugins/jenkins-pipeline.yaml

```diff
@@ -9,43 +9,43 @@ tools:
   options:
     jenkins:
       # url is used to config jenkins url
-      url: http://jenkins.example.com:8080
```

Comment:
If we change all the example configurations to placeholders, the complexity of using them will be much higher. The same configuration file would require more modifications.